helm-chart/dash0-operator/README.md

```diff
@@ -727,6 +727,36 @@ resource attribute auto-detection.
 This allows Dash0 users to avoid the hassle of manually adding the OpenTelemetry SDK to their applications.
 Dash0 takes care of it automatically.
 
+### Specifying additional resource attributes via pod labels and annotations
```

Comment:
I don't think this section should be here in the "This section provides a quick behind-the-scenes glimpse into how the Dash0 operator's workload instrumentation for tracing works" part of the README. 

The fact that we implement this configurability via the injector is a technical detail, the option to set the resource attributes are an important feature though. It needs to be in the "Configuration" main section, starting in line 126. There is no good sub-section to add it there yet, we should probably create a new sub-section called something like "Per-workload configuration" or something similar and put this new content there, alongside with what we have now in "Disabling Auto-Instrumentation for Specific Workloads".

Reply:
Done in a4cc1d3